test-server/test-echo.c

```diff
@@ -189,7 +192,7 @@ int main(int argc, char **argv)
 	struct libwebsocket_context *context;
 	int opts = 0;
 	char interface_name[128] = "";
-	const char *interface = NULL;
+	const char *_interface = NULL;
```

Comment:
This 'interface' is a msft perversion, they seem to #define it to 'struct' in the headers which is just as unfriendly as you'd expect from them.

So if we can't exclude the header that does the dirty, or #undef interface in private-libwebsockets.h,  changing 'interface' in the code is reasonable.